tournaments/index.js

```diff
@@ -794,6 +794,7 @@ var commands = {
 			if (params.length < 1) {
 				return this.sendReply("Usage: " + cmd + " <minutes|off>");
 			}
+			if (params[0] === 'Infinity' || params[0] === '0') params[0] = 'off';
```

Comment:
The entire point of this was to prevent the command from entering ``(The tournament auto disqualify timeout was set to Infinity.)`` in the modlog. Changing ``Infinity`` to ``off``, which already uses Infinity to stop the timeout, fixes this.